components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/add-wf-wizard.jsp

```diff
@@ -167,7 +167,7 @@
 
             var workflowId;
             <% if (workflowId != null) { %>
-            workflowId = "<%=workflowId%>";
+            workflowId = "<%=Encode.forJavaScript(workflowId)%>";
```

Comment:
Since this is inside <script> block, it's correct to use Encode.forJavascript [1]

[1] https://owasp.github.io/owasp-java-encoder/encoder/apidocs/org/owasp/encoder/Encode.html#forJavaScript-java.lang.String-